tests/test_price_parsing.py

```diff
@@ -1901,13 +1901,13 @@ def __eq__(self, other):
             'CHF', '19.90', 19.90),
     Example('', '530,42 Zł',
             'Zł', '530,42', 530.42),
+    Example('3 Ausgaben für nur 14,85 EUR', '3 Ausgaben für nur 14,85 EUR',
+            'EUR', '14,85', 14.85),
 ]
 
 
 PRICE_PARSING_EXAMPLES_XFAIL = [
     # amount is picked as a price
-    Example('3 Ausgaben für nur 14,85 EUR', '3 Ausgaben für nur 14,85 EUR',
-            'EUR', '14,85', 14.85),
     Example(None, 'Buy Now - 2 Litre Was $120.00 Now $60.00',
```

Comment:
I wonder why isn't this example fixed?

Reply:
Because `120.00` is picked instead of `60.00`.

I believe this test is only meant to pass once price-parser understand how to prefer current prices over previous prices when a string has both ("was" vs "now").